src/ngraph/frontend/onnx_import/core/null_node.hpp

```diff
@@ -25,6 +25,7 @@ namespace ngraph
 {
     namespace onnx_import
     {
+        /// \brief Placeholder for expected but missing node in graph.
```

Comment:
This is kind of misleading. We won't put anything in place of this node, are we? If it is expected then decidedly it is an error if it is missing.

Reply:
Please use a docstring such as this:

> Represents a missing optional input or output of an ONNX node
> 
> Some ONNX operators have inputs or outputs that are marked as optional, which means that a referring node MAY forgo providing values for such inputs or computing these outputs. An empty string is used in place of a name of such input or output.
> 
> See:
> https://github.com/onnx/onnx/blob/master/docs/IR.md#optional-inputs-and-outputs
>